src/command_modules/azure-cli-backup/HISTORY.rst

```diff
@@ -3,6 +3,10 @@
 Release History
 ===============
 
+1.0.3
++++++
+* Refactored code to use parse_id tools from msrestazure.
```

Comment:
Since this is refactoring, these are not "customer-facing, functional" changes and really offer value in the history notes. They should be removed.

Reply:
There needs to be a version update since there is code being changed. Maybe "* minor fixes" instead of statement the code change would be more appropriate?

Reply:
Yeah, minor fixes works :)